README.md

```diff
@@ -1,4 +1,51 @@
-# nswag-extensions
-Nswag extension to support query string versioning
+# Ibistic.Public.NSwag.Extensions
+A framework that provides extensions to create OpenAPI 3.0 specifications with NSwag.
 
-Pending to add info to the readme file.
+NSwag toolchain used from https://github.com/RicoSuter/NSwag under the license https://github.com/RicoSuter/NSwag/blob/master/LICENSE.md.
+
+Please only use this component if you agree to and comply with the above license terms.
+
+## Install
+
+To install Ibistic.Public.NSwag.Extensions, run the following command in the Package Manager Console
+
+    PM> Install-Package Ibistic.Public.NSwag.Extensions
+
+## Documentation
+
+Ibistic.Public.NSwag.Extensions has by now one new api version processor.
```

Comment:
Ibistic.Public.NSwag.Extensions currently only has a processor for including api version in the query string.

Reply:
Done.